scripts/rich-core-check-oneshot

```diff
@@ -8,20 +8,26 @@ IFS=$'\n'
 USERID=$(loginctl list-sessions | grep seat0 | tr -s " " | cut -d " " -f 3)
 ONESHOTS="$(find -L $(find /etc/oneshot.d/$USERID -type d) -maxdepth 1 -type f)"
 
-PREVIOUS_ONESHOTS="$(cat $ONESHOTS_FILE 2>/dev/null)"
+PREVIOUS_ONESHOTS=$(mktemp /tmp/previous_oneshots.XXXXXX)
+cat $ONESHOTS_FILE >> $PREVIOUS_ONESHOTS
 
 echo "$ONESHOTS" | while read oneshot; do
-	is_new=0
-	echo "$PREVIOUS_ONESHOTS" | while read oldshot; do
+	is_new=1
+	while read oldshot; do
```

Comment:
Now that this uses a file, this is basically grepping the file for $oneshot line. Something like grep -x "$oneshot" "$PREVIOUS_ONESHOTS" should do it, I think.

Reply:
Well we could do this also with grep, but problem is the case where find returns null. Then we should check if oneshots file is empty or not and update file if needed or update file all ways with the results from find operation. I don't know if it makes script any better or simpler.